project/src/test/java/com/google/sps/VolunteerOpportunityTest.java

```diff
@@ -0,0 +1,77 @@
+package com.google.sps;
+
+import com.google.common.collect.ImmutableSet;
+import com.google.sps.data.VolunteeringOpportunity;
+import java.util.Set;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** */
+@RunWith(JUnit4.class)
+public final class VolunteerOpportunityTest {
+  private static VolunteeringOpportunity opportunity;
```

Comment:
I think it would be good for each test to be as independent of each other as possible, so opportunity would be instantiated within each test (correct me if I'm wrong, but since the class is mutable, there could be issues if the tests did something more fancy)

Reply:
Done - thanks right especially because class is mutable.